services/build.gradle

```diff
@@ -6,14 +6,15 @@ sourceSets {
                          '../services-optimization/src/main/java', '../services-staticmap/src/main/java',
                          '../services-speech/src/main/java', '../services-tilequery/src/main/java',
                          '../services-route-tiles/src/main/java', '../services-directions-refresh/src/main/java',
-                         '../services-isochrone/src/main/java']
+                         '../services-isochrone/src/main/java', '../services-matching/src/main/java']
 }
 
 dependencies {
     api project(":services-core")
     api project(":services-geojson")
     api project(":services-directions-models")
     api project(":services-directions-refresh-models")
+    api project(":services-matching")
```

Comment:
Why are we adding this but `services-isochrone` or `services-speech` (to name a couple) are not in here? 🤔

Reply:
removed, it's not affecting docs

Reply:
At some point we should revisit this (what we're including as part of `services`). Could you cut a ticket when you have a chance so we don't forget? 🙏

Reply:
done https://github.com/mapbox/mapbox-java/issues/1401

Reply:
Thanks 🙇‍♂️